### PR TITLE
Do not search orgchart for contributors.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: mozillians/dino_park/views.py [hunk 1]
```diff
@@ -47,6 +47,12 @@ def orgchart_get_by_username(request, path, username):
         user = User.objects.get(username=username)
     except User.DoesNotExist:
         user = None
+
+    # if there is a user and the user is not a staff member
+    # then we don't need to search for a profile in orgchart
+    if user and not user.userprofile.is_staff and path == 'trace':
+        return JsonResponse(None, safe=False)
```
Comment (johngian) [inline mozillians/dino_park/views.py]: why `safe=False` ? Shouldn't it be handles by `JsonResponse` ?

File: mozillians/dino_park/views.py [hunk 2]
```diff
@@ -47,6 +47,12 @@ def orgchart_get_by_username(request, path, username):
         user = User.objects.get(username=username)
     except User.DoesNotExist:
         user = None
+
+    # if there is a user and the user is not a staff member
+    # then we don't need to search for a profile in orgchart
+    if user and not user.userprofile.is_staff and path == 'trace':
```
Comment (johngian) [inline mozillians/dino_park/views.py]: i am not sure but maybe ``trace`` should be ``/trace``

File: mozillians/dino_park/views.py [hunk 3]
```diff
@@ -61,7 +67,6 @@ def orgchart_get_by_username(request, path, username):
     )
     url = urlparse.urlunparse(url_parts)
     resp = requests.get(url)
-    resp.raise_for_status()
```
Comment (johngian) [inline mozillians/dino_park/views.py]: what happens if response is not ``20x`` or ``30x`` ?

File: mozillians/dino_park/views.py [hunk 4]
```diff
@@ -104,5 +109,4 @@ def search_get_profile(request, username, scope=None):
...
```
Comment (johngian) [inline mozillians/dino_park/views.py]: Same here

File: mozillians/dino_park/views.py (hunk 3 above)
Comment (akatsoulas) [reply mozillians/dino_park/views.py]: These errors should eventually be handled in the frontend

File: mozillians/dino_park/views.py (hunk 1 above)
Comment (akatsoulas) [reply mozillians/dino_park/views.py]: Without that the `data` payload should be only dictionary.

File: mozillians/dino_park/views.py (hunk 2 above)
Comment (akatsoulas) [reply mozillians/dino_park/views.py]: `trace` should be fine. Slashes are added during the url construction

**Pull request merged**